spec/policy_spec.lua

```diff
@@ -49,6 +49,56 @@ describe('policy', function()
     end)
   end)
 
+  describe('module equality', function()
+    it('equals when name and version are the same', function()
+      local p1 = policy.new('NAME', 'VERSION')
+      local p2 = policy.new('NAME', 'VERSION')
+
+      assert.are.equal(p1, p2)
+      assert.not_same(p1, p2)
+    end)
+
+    it('is not equal when names are different', function()
+      local p1 = policy.new('NAME', 'VERSION')
+      local p2 = policy.new('NAME2', 'VERSION')
+
+      assert.are.not_equal(p1, p2)
+    end)
+
+    it('is not equal when versions are different', function()
+      local p1 = policy.new('NAME', 'VERSION')
+      local p2 = policy.new('NAME', 'VERSION2')
+
+      assert.are.not_equal(p1, p2)
+    end)
+  end)
+
+  describe('module tostring', function()
+    it('shows name and version', function()
+      local p1 = policy.new('NAME', 'VERSION')
+
+      assert.equal('Policy: NAME (VERSION)', tostring(p1))
+    end)
+  end)
+
+  describe('instance equality', function()
+    local p1 = policy.new('NAME', 'VERSION')
+    local p2 = policy.new('NAME', 'VERSION')
+
+    it('equals when name and version are the same', function()
+      assert.are.equal(p1.new(), p1.new())
+      assert.are.equal(p1.new(), p2.new())
+    end)
+  end)
+
+  describe('instance tostring', function()
+    it('shows name and version', function()
+      local p1 = policy.new('NAME', 'VERSION').new()
+
+      assert.equal('Policy: NAME (VERSION)', tostring(p1))
```

Comment:
This looks a bit strange to me because the policy module and the instances have the same string representation.

Reply:
Yeah, but I have no good ideas how to differentiate them. This is useful when passing them to inspect and there you can kind of see if it is an instance or not.

Reply:
We could include the config in the instances. Although it might be a very big table depending on the policy.

Reply:
Hmm now that I think about it, we don't have a `self.config` for each policy.
I guess that for now this is good enough. As you said, with inspect we'll be able to cleary differentiate them.